packages-next/admin-ui/src/templates/adminMetaSchemaExtension.ts

```diff
@@ -95,9 +95,11 @@ export function adminMetaSchemaExtension({
   graphQLSchema: GraphQLSchema;
   isAccessAllowed: undefined | ((args: { session: any }) => boolean | Promise<boolean>);
 }) {
-  const lists: StaticAdminMetaQueryWithoutTypeNames['keystone']['adminMeta']['lists'] = Object.values(
-    adminMeta.lists
-  ).map(({ gqlNames, fields, ...list }) => {
+  type T = StaticAdminMetaQueryWithoutTypeNames['keystone']['adminMeta'];
```

Comment:
T is confusing as T is used usually in the type parameter

Reply:
Agree, I'd go with the `AdminMeta` suggestion